src/Sylius/Behat/Context/Setup/PaymentRequestContext.php

```diff
@@ -42,4 +50,37 @@ public function thePaymentRequestActionHasBeenExecutedForOrderWithThePaymentMeth
 
         $this->commandBus->dispatch($addPaymentRequest);
     }
+
+    /**
+     * @Given there is (also) a payment request action :action executed for order :order with the payment method :paymentMethod and state :state
```

Comment:
```suggestion
     * @Given there is (also) a :state :action payment request for order :order using the :paymentMethod payment method
```
So we can just write
`Given there is a new authorize payment request for order ...`